src/quo2/components/tags/scrollable_view.cljs

```diff
@@ -0,0 +1,146 @@
+                                                                    layout-width   (oget e "nativeEvent.layoutMeasurement.width")
+                                                                    new-percentage (calculate-fade-end-percentage {:offset-x            offset-x
+                                                                                                                   :content-width       content-width
+                                                                                                                   :layout-width        layout-width
+                                                                                                                   :max-fade-percentage fade-end-percentage})]
+                                                                ;; Avoid unnecessary re-rendering.
+                                                                (when (not= new-percentage (get @fading :fade-end-percentage))
+                                                                  (swap! fading assoc :fade-end-percentage new-percentage))))
+                                                            (when on-scroll
+                                                              (on-scroll e)))
+                       :render-fn                          (fn [{:keys [id label resource]} index]
+                                                             [rn/view {:style {:margin-right  (if (= size default-tab-size) 12 8)
+                                                                               :padding-right (when (= index (dec (count data)))
+                                                                                                (get-in props [:style :padding-left]))}}
+                                                              (if (= component :tags)
+                                                                [tag/tag {:id            id
+                                                                          :size          size
+                                                                          :active        (= id @active-tab-id)
+                                                                          :resource      resource
+                                                                          :blurred?      blurred?
+                                                                          :icon-color    icon-color
+                                                                          :disabled?     disabled?
+                                                                          :label         label
+                                                                          :type          type
+                                                                          :labelled?     labelled?
+                                                                          :on-press      (fn [id]
```

Comment:
The `on-press` anonymous functions look identical. If this is true duplication, we could at least extract this function to a let binding.